cmd/monitor/collector/bootstrap.yaml

```diff
@@ -30,7 +30,7 @@ erda.oap.collector.core:
     metrics:
       - receivers:
           #- "erda.oap.collector.receiver.dummy"
-          - "erda.oap.collector.receiver.prometheus-remote-write"
+          - "erda.oap.collector.receiver.prometheus-remote-write@prometheus"
```

Comment:
Do we not need to adjust the collector-agent?

Reply:
not need